conduit/src/Data/Conduit/Internal/Conduit.hs

```diff
@@ -989,6 +990,20 @@ mapInput f f' (ConduitT c0) = ConduitT $ \rest -> let
     go (Leftover p i) = maybe id (flip Leftover) (f' i) (go p)
     in go (c0 Done)
 
+-- | Apply a monadic action to all the input values of a @ConduitT@.
```

Comment:
Please add a `@since` comment, ChangeLog entry, and minor version bump. Otherwise looks good, thanks!

Reply:
Done!

PS: This is the fastest review for a PR I've ever had 😸